platform.json

```diff
@@ -59,6 +64,11 @@
       "type": "uploader",
       "optional": true,
       "version": "^1.63208.0"
+    },
+    "tool-nrfutil": {
```

Comment:
I propose to remove this package at all. Linux user should install it manually because it is not Python 2/ Python 3 compatible.